### ISSUE TITLE
Fi code in parallel and parameters ?

### ISSUE THREAD

**Issue by Timothe20** [amusecode/amuse#191]
Hi,
I want to simulate a star forming region. I am making some simple examples with the code Fi such as the collapse of a molecular cloud. But I have some problems to make it work in parrallel to know its ability to be used for more complex cases (it seems it need the OMP_NUM_THREADS environment variable to be set...). The build log just tells me fi-gl was not built because it needs special libraries (which ? and I have to admit that I do not know what "gl" means).

Another thing, I find very difficult to understand and tune the different parameters of Fi, is there any summary of what are the meaning of all these parameters ? I would prefer not to go deeply in the low level code.

**Comment by spzwart** [comment]
dear Timothe,

maybe the book helps, check out my website under 'mission', where you will find a pre-publication copy of the manuscript. you answer can be found in chapter 5 (hydrodynamics) and in appendix B you find more info on Fi.

Groeten,
Simon

----- Reply message -----

**Comment by rieder** [comment]
Hi @Timothe20,
fi-gl is a version of fi with OpenGL support, it is not needed for running SPH calculations. It requires the f90gl library which does not compile on all systems, most notably macOS. You can safely ignore it and use fi without OpenGL support.

**Comment by Timothe20** [comment]
Ok , I see, thanks. And how about running the code in parallel ? I am using openmpi which work fine for gravitationnal dynamics codes.

**Comment by rieder** [comment]
Fi does not support multiple workers through MPI, it works with OpenMP though. 
Run Fi with number_of_workers=1, mode='openmp' and set the OMP_NUM_THREADS environment variable to the number of processors you'd like to use.

**Comment by rieder** [comment]
Closing the issue, but please feel free to re-open if something doesn't work as expected!
